README.md

```diff
@@ -16,6 +16,9 @@ You can contact us for support on our [chat server](https://discord.gg/V3vSCs7)
 ```js
 const fs = require('fs');
 const ytdl = require('ytdl-core');
+// TypeScript: import ytdl from 'ytdl-core'; with --esModuleInterop
+// TypeScript: import * as ytdl from 'ytdl-core'; with --allowSyntheticDefaultImports
+// TypeScript: import ytdl = require('ytdl-core'); with neither of the above
```

Comment:
I'm not entirely sure why we would include every example of TypeScript. Surely if they were using TypeScript they'd know how to import a module no?